File: C/String/program7.c
//Display the string

#include<stdio.h>

void Display(char str[])
{
    printf("The string is: \n");
    
    while(*str != '\0')
    {
        printf("%c",*str);
        str++;
    }
}

int main()
{
    char Arr[20];

    printf("Enter string: \n");
    scanf("%[^'\n']s",Arr);

    Display(Arr);

    return 0;
}

File: Java/String/program7.java
//Accept the string from user and display its length

import java.util.*;

class Stringx
{
    public String str;

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the string");
        str = sobj.nextLine();
    }

    public int strlenX()
    {
        char Arr[] = str.toCharArray();

        int iCnt = 0;

        iCnt = Arr.length;

        return iCnt;
    }
}

class program7
{
    public static void main(String arg[])
    {
        int iRet = 0;

        Stringx obj = new Stringx();

        obj.Accept();

        iRet = obj.strlenX();
        System.out.println("String length is : "+iRet);
    }
}